### PR TITLE
Vuepress feedback

### PR DESCRIPTION
**Pull request by abea**

### CODE REVIEW

File: docs/.vuepress/theme/components/AposFeedback.vue [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  methods: {
+    sendFeedback: function(evtType) {
+      this.voteSubmitted = true;
+      console.log(evtType);
+
+      if (!window.ga) {
+        return;
+      }
+
+      window.ga('send', 'event', {
```
Comment (boutell) [inline docs/.vuepress/theme/components/AposFeedback.vue:42]: YAAAS for using google analytics as a model/reporting layer but rolling our own dang frontend and behavior to get this The Way We Want.

Comment (abea) [reply docs/.vuepress/theme/components/AposFeedback.vue:42]: Just wait until I add the textarea input...

File: docs/.vuepress/theme/components/AposFeedback.vue [hunk 2]
```diff
@@ -0,0 +1,162 @@
+<template>
+  <!-- Based on https://github.com/ipfs/ipfs-docs-v2/blob/2a019d7c8abe99c21a3150752915ed278a4592ff/docs/.vuepress/theme/components/Feedback.vue -->
```
Comment (abea) [inline docs/.vuepress/theme/components/AposFeedback.vue:2]: This is mostly all courtesy of the `ipfs-docs-v2`, though I changed presentation and removed a few bits we don't need. If any seems overly superfluous, we can remove it, but we'll also likely be doing cosmetic work later on anyway.

File: docs/.vuepress/theme/components/AposFeedback.vue [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    },
+    noTxt: {
+      type: String,
+      default: 'No'
+    }
+  }
+}
+</script>
+
+<style lang="stylus" scoped>
```
Comment (abea) [inline docs/.vuepress/theme/components/AposFeedback.vue:87]: vuepress uses stylus.

File: docs/.vuepress/theme/components/AposFeedback.vue [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+  @media (min-width: $MQMobileNarrow) {
+    display: block;
+
+    .feedback-button:last-child {
+      margin-top: 0;
+    }
+  }
+}
+
+.feedback-button {
```
Comment (abea) [inline docs/.vuepress/theme/components/AposFeedback.vue:128]: The button from the Apostrophe website.

File: docs/.vuepress/theme/components/Page.vue [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+<style lang="stylus">
+@require '../styles/wrapper.styl';
+
+.page {
+  padding-bottom: 2rem;
+  display: block;
+}
+
+.page-footer {
+  @extend $wrapper;
```
Comment (abea) [inline docs/.vuepress/theme/components/Page.vue:42]: that's from the Vuepress default theme.

**Comment by stuartromanek** [comment]
Some random bits and suggestions

**Logo gets obscured quickly**
image

**CSS fiddling, obviously normalize units**
```css
.navbar a {
    	font-size: 12px;
    	font-weight: 500;
    }

    .sidebar > .sidebar-links > li > a.sidebar-link,
    .sidebar .sidebar-sub-headers {
    	font-size: 0.9em
    }

    h2 {
    	font-size: 24px;
    	font-weight: 600;
    }
```

**Helpful? Design**
image

**Comment by abea** [comment]
Thanks. I'll add the logo bit as a new ticket to do, but will get the helpful update in.

**Comment by abea** [comment]
Updated on staging https://docs.apos.dev/devops/

**Pull request merged**